### ISSUE TITLE
super[]() effect problem

### ISSUE THREAD

**Issue by Lotzyprod** [Pesekjak/Hippo#3]
Hello! I found one problem that confuses me

Hippo class:
```
public skript class crew.nms.DummyPlayerConnection extends PlayerConnection:
    public DummyPlayerConnection(MinecraftServer minecraftServer, NetworkManager networkManager, EntityPlayer entityPlayer):
        super[MinecraftServer,NetworkManager,EntityPlayer]({_minecraftServer}, {_networkManager}, {_entityPlayer})
```
```
set {_playerConnection} to new DummyPlayerConnection({_minecraftServer},{_networkManager}, {_entityPlayer})
# throw null pointer exception:

# constructor DummyPlayerConnection#<init> called with (net.minecraft.server.dedicated.DedicatedServer@6d123710 (DedicatedServer),
# net.minecraft.network.NetworkManager@145883af (NetworkManager), EntityPlayer['test'/1, uuid='f34b1200-30ea-4198-ae93-08d58c182326',
# l='ServerLevel[.world]', x=0.50, y=68.00, z=0.50, cpos=[0, 0], tl=0, v=false](test at 0.5,68.0,0.5) (EntityPlayer)) threw a NullPointerException:
# Cannot invoke "net.minecraft.network.NetworkManager.a(net.minecraft.network.PacketListener)" because "connection" is null.

set {_playerConnection} to new PlayerConnection({_minecraftServer},{_networkManager}, {_entityPlayer})
# not throw 
```

**Comment by Lotzyprod** [comment]
upd. anyway check forks, and in this fork https://github.com/msgdw/Hippo problem doesn't exists

**Comment by Pesekjak** [comment]
Thanks for reporting, I know about this issue, I plan on fixing it in the next update

**Comment by envizar** [comment]
> Thanks for reporting, I know about this issue, I plan on fixing it in the next update

When?

**Comment by Pesekjak** [comment]
New version for 2.7 is almost finished, I am actually looking for testers right now, so if you would be interested you can message me on discord.

**Comment by envizar** [comment]
> New version for 2.7 is almost finished, I am actually looking for testers right now, so if you would be interested you can message me on discord.

Can you just publish it as alpha release?

**Comment by Pesekjak** [comment]
fixed in #5
